business ID visibility, filtering, and correlation follow up (#9570)
camunda/camunda-docs: docs/reference/announcements-release-notes/8100/8100-release-notes.md

<p class="link-arrow">[Feature flag reference](/self-managed/components/hub/configuration/properties.md#hide-invite-member-button)</p>

### Operate

#### Business ID visibility for decision instances

<!-- https://github.com/camunda/product-hub/issues/3436 -->

<div class="release"><span class="badge badge--medium" title="This feature affects Operate">Operate</span></div>

Business ID is now visible in Operate for decision instances, in both the decision instance list and the decision instance details view. Filter decision instances by business ID using **Equals**, **Contains**, and **Is one of** in the filter UI, or the full operator set (`$eq`, `$neq`, `$exists`, `$like`, `$in`, `$notIn`) via the API.

<p class="link-arrow">[Business ID](/components/operate/userguide/basic-operate-navigation.md#business-id-for-decision-instances)</p>

### Orchestration Cluster

#### FEEL context variables for the process instance

<!-- https://github.com/camunda/product-hub/issues/3436 -->

<div class="release"><span class="badge badge--medium" title="This feature affects Orchestration Cluster">Orchestration Cluster</span></div>

> The process instance is now accessible in FEEL expressions via the `camunda.processInstance` context, resolvable anywhere in the process — not only in call activity expressions. `camunda.processInstance.key` returns the process instance's system-generated key, and `camunda.processInstance.businessId` returns its business ID (or `null` if none is set).

Reviewer: ```suggestion
The process instance properties are now accessible in FEEL expressions via the `camunda.processInstance` context, resolvable anywhere in the process. `camunda.processInstance.key` returns the process instance's system-generated key, and `camunda.processInstance.businessId` returns its business ID (or `null` if none is set).
```
